modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -396,9 +396,13 @@ State:                      ready
 Private:                    No
 Created:                    Oct  1 2021 08:12:25 UTC
 Details Page:               https://console.redhat.com/openshift/details/s/<subscription_id>
-OIDC Endpoint URL:          https://rh-oidc.s3.<aws_region>.amazonaws.com/<cluster_id>
+OIDC Endpoint URL:          https://oidc.p1.openshiftapps.com/<cluster_id>|<odic_config_id \ <1>
 ----
 +
+--
+1. The endpoint URL depends on the BYO OIDC configuration. If you are bringing your own OIDC, the URL should end with `<oidc_config_id>`; otherwise, it should be the `cluster-ID`.
```

Comment:
```suggestion
1. The endpoint URL depends on the BYO OIDC configuration. If you are pre-creating the OIDC configuration, the URL should end with `<oidc_config_id>`; otherwise, it should be the `<cluster_id>`.
```